capstone/src/main/java/com/google/step/clients/MetricsClient.java

```diff
@@ -140,13 +142,14 @@ public WeeklyPageView getCurrentPageViews(String restaurantKey) {
                 List<WeeklyPageView> currRestaurantList = new ArrayList<>();
                 currRestaurantList.add(new WeeklyPageView(week, year, count));
                 restaurantNameMap.put(name, currRestaurantList);
```

Comment:
Is it safe to assume that restaurant name is unique?

I'm okay with it for demo purposes, if you want to back-burner it.

Reply:
Fixed so it utilized id instead which is unique per restaurant.